grunttasks/build.js

```diff
@@ -14,7 +14,6 @@ module.exports = function (grunt) {
 
     if (target === 'development') {
       taskArr = [
-        'lint',
```

Comment:
One of my primary use cases is: something's busted, let's insert a `debugger` statement and step through it with node-debugger. But the linter complains every time, and every time I disable it temporarily in the gruntfile. Might as well optimize for actual _development_ on the server.